datacontract.schema.json

```diff
@@ -272,21 +272,51 @@
                 "type": "string",
                 "description": "An optional string describing the server."
               },
+              "host": {
+                "type": "string",
+                "description": "An optional string describing the host name."
+              },
               "database": {
                 "type": "string",
                 "description": "An optional string describing the server."
               },
               "schema": {
                 "type": "string",
                 "description": "An optional string describing the server."
+              },
+              "clusterIdentifier": {
+                "type": "string",
+                "description": "An optional string describing the cluster's identifier.",
+                "examples": [
+                  "redshift-prod-eu",
+                  "analytics-cluster"
+                ]
+              },
+              "port": {
+                "type": "integer",
+                "description": "An optional string describing the cluster's port.",
+                "examples": [
+                  5439
+                ]
+              },
+              "endpoint": {
+                "type": "string",
+                "description": "An optional string describing the cluster's endpoint.",
+                "examples": [
+                  "analytics-cluster.example.eu-west-1.redshift.amazonaws.com:5439/analytics"
+                ]
               }
             },
             "additionalProperties": true,
             "required": [
               "type",
               "account",
+              "host",
               "database",
-              "schema"
+              "schema",
+              "clusterIdentifier",
```

Comment:
Are really all required? This would not match your examples in the README.

Reply:
@jochenchrist Apologies for the mistake – I’ve now corrected that oversight and tidied up my examples in the README to be consistent.